content/articles/using-logistic-regression-in-pytorch-to-identify-handwritten-digits/index.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /using-logistic-regression-in-pytorch-to-identify-handwritten-digits/
+title: Using Logistic Regression in PyTorch to Identify Handwritten Digits
+description: 
+author: masese-gavin
+date: 2021-11-19T00:00:00-18:00
+topics: [Machine Learning]
+excerpt_separator: <!--more-->
+images:
+
+ - url: /engineering-education/implementing-gan-from-scratch/hero.jpg
+   alt: Implementing GANs example image
+---
+
+### Introduction
+Logistic Regression is a widely used statistical method for predicting a binary outcome from a set of independent variables. This [article](https://www.geeksforgeeks.org/understanding-logistic-regression/) previously covered the various properties of logistic regression and its Python implementation. Now we'll look at how to implement this in PyTorch and use Logistic Regression to classify handwritten digits from the MNIST dataset.
+
+### Prerequisites
+1. Install [PyTorch](https://pytorch.org/) into your Python environment.
+2. Having some knowledge of the Python programming language is a good idea.
+
+### Table of contents
+- [Library functions and objects](#library-functions-and-objects)
+- [Defining our hyperparameters](#defining-our-hyperparameters)
+- [Building The Neural Network](#building-the-neural-network)
+- [References ](#references )
+- [Conclusion](#conclusion)
+ 
+### Library functions and objects
+Importing library functions and objects into your code is necessary once your installation is complete.
+Here are the functions and objects to import:
+
+- The **torch.nn** module: Contains code thatwe will be using in our model.
+- The **torchvision.datasets**: Includes MNIST dataset of handwritten digits that we shall be using here.
+- The **torchvision.transforms**: We shall be using it to transform from images to PyTorch tensors.
+- The **torch.autograd**: This will be used to define our tensors.

```

Comment:
please check the functionality of autograd. It is not used to define the tensors.